web/src/api/helpers.ts

```diff
@@ -98,6 +98,7 @@ export function isValidDate(dateString: string) {
 export const TIME_RANGE_TO_TIME_AVERAGE: Record<TimeRange, string> = {
   [TimeRange.H72]: 'hourly',
   [TimeRange.M3]: 'daily',
+  [TimeRange.M12]: 'monthly',
```

Comment:
😅 
![Screenshot 2025-01-09 at 16 42 21](https://github.com/user-attachments/assets/ada11381-28e9-4fc3-90e5-6c0713f2011c)

Reply:
Kind of by design as I need to re-adjust the app backend again, PR incoming!

Reply:
Backend PR is deploying now so this should be fixed momentarily.